crates/ads126x/src/register.rs

```diff
@@ -0,0 +1,94 @@
+use bitflags::bitflags;
+
+#[repr(u8)]
```

Comment:
`#[repr(u8)]` should prevent any non-u8 `Register` values at comptime which increases the safety of casting this to u8 for RREG and WREG.

Reply:
Nice!